core-httpclient-impl/src/main/java/com/optimizely/ab/OptimizelyFactory.java

```diff
@@ -179,13 +179,29 @@ public static Optimizely newDefaultInstance(String sdkKey) {
      * @param fallback Fallback datafile string used by the ProjectConfigManager to be immediately available.
      */
     public static Optimizely newDefaultInstance(String sdkKey, String fallback) {
+        String datafileAuthToken = PropertyUtils.get(HttpProjectConfigManager.CONFIG_DATAFILE_AUTH_TOKEN);
```

Comment:
We've been using `datafileAccessToken` in the other SDKs. Let's stick with the same name throughout.